src/main/java/quickcarpet/commands/CameraModeCommand.java

```diff
@@ -53,8 +53,10 @@ private static int cameraMode(ServerCommandSource source, ServerPlayerEntity tar
         if (!(hasPermission(source, target)) || target.isSpectator()) return 0;
         QuickCarpetServer.getInstance().cameraData.put(target.getUuid(), new CameraData(target));
         target.changeGameMode(GameMode.SPECTATOR);
-        target.addStatusEffect(new StatusEffectInstance(StatusEffects.NIGHT_VISION, 999999, 0, false, false));
-        target.addStatusEffect(new StatusEffectInstance(StatusEffects.CONDUIT_POWER, 999999, 0, false, false));
+        if(quickcarpet.settings.Settings.cameraModeNightVision) {
```

Comment:
You'd only need the fully qualified name of Settings in Block mixins for example, but doesn't matter